object/permission_enforcer.go

```diff
@@ -35,7 +39,7 @@ func getEnforcer(permission *Permission) *casbin.Enforcer {
 r = sub, obj, act
 
 [policy_definition]
-p = permission, sub, obj, act
+p = sub, obj, act
```

Comment:
Thank you @seriouszyx In one of the screens you have shared earlier for the RBAC + domains model, the policy looked like having 4 variables 

![image](https://user-images.githubusercontent.com/4610831/184790621-620be2ef-2a68-4c15-b52a-d146f7d27efa.png)

Wondering about this `p = sub, obj, act` since it seems to be lacking the domain vairable

Reply:
In order to prevent the program from crashing due to the model in permission being empty, I set a default model (RBAC model) by hard coding. If using the `RBAC with domains` model, you can add a new model and assign it to permission. 

As for the `permission` column (p = permission, sub, obj, act) in the policy definition, we previously set it to distinguish between different permissions in the same database table. But now we support the `adapter` configuration to specify the table name for policy storage. So the `permission` column is unnecessary.  

After this PR is merged, I will refactor the permission documentation and explain the exposed casbin APIs (https://github.com/casdoor/casdoor-website/issues/272).

Reply:
@KrishnaPG